README.md

```diff
@@ -3,18 +3,18 @@
 This service enables postgraduate candidates to apply for initial teacher
 training.
 
-## Initial setup
+## Prerequisites
 
-Clone this repo and run `bundle install`.
+- Ruby 2.6.3
+- NodeJS 8.11.x
+- Yarn 1.12.x
```

Comment:
Also, why do we require yarn; does it offer something npm doesn’t?

Reply:
I think `yarn` v `npm`'s just a preference - I don't have any strong feelings one way or the other so long as we're consistent. Not sure which is currently considered "better".

Reply:
I’m sure Yarn has it’s benefits, and I’m no expert, but as I recall, the things it was created to solve have been since been added to npm. I guess the main reasons for not using Yarn is that it’s an additional dependancy and something else to manage. And well, the project’s provenance doesn’t help, either. 😉